### ISSUE TITLE
Error: The setter 'textScaler' isn't defined for the class 'RenderScatterChart'.

### ISSUE THREAD

**Issue by hamzakircicek** [imaNNeo/fl_chart#1509]
I'm using 3.16.0 version. I have this issue.

**Comment by techouse** [comment]
I get the same with 3.13.9.
```
Error (Xcode): ../../.pub-cache/hosted/pub.dev/fl_chart-0.65.0/lib/src/chart/bar_chart/bar_chart_renderer.dart:42:5: Error: Type 'TextScaler' not found.
```

**Comment by ishanSiterecon** [comment]
I am facing a similar issue, I tried a custom widget in Flutterflow. I was using the "import 'package:fl_chart/fl_chart.dart';"  package and it worked fine but since yesterday or so, I have been facing issues in deploying the same custom widget which worked earlier. 

 errors:
 "bar_chart_renderer.dart:42:5: Error: Type 'TextScaler' not found.
    TextScaler textScaler,"
    "line_chart_renderer.dart:75:18: Error: Type 'TextScaler' not found.
  set textScaler(TextScaler value) {"
  
  "radar_chart_renderer.dart:74:18: Error: Type 'TextScaler' not found.
  set textScaler(TextScaler value) {"
  
  It seems like a library dependency issue because it is trying to search textScaler in even charts I am not using like radar. Also, it was working earlier for me. I am using Flutter 3.17.7 Flutterflow v4.0 (above suggestion did not helped for me)

**Comment by imaNNeo** [comment]
> @imaNNeo a Flutter/Dart SDK version constraint should probably be added to 0.65.0 to avoid this issue, i.e.
> 
> ```yaml
> environment:
>   sdk: ^3.2.0
>   flutter: ^3.16.0
> ```

You're right. I forgot to do that.
Sorry about that guys, please downgrade your fl_chart version to the previous version, or upgrade your flutter to the latest version to use the latest fl_chart version.

**Comment by imaNNeo** [comment]
Fixed in the [0.66.0](https://github.com/imaNNeo/fl_chart/releases/tag/0.66.0) release.
